src/components/Missions.js

```diff
@@ -1,7 +1,16 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import fetchMissions from '../helpers/fetchMissions';
+import { getMissions } from '../redux/missions/missions';
+import store from '../redux/storeConfig';
 
-const Missions = () => (
-  <div>Missions</div>
-);
+const Missions = () => {
+  useEffect(async () => {
+    store.dispatch(getMissions(await fetchMissions()));
+  }, []);
+
```

Comment:
[Required]
Please move useEffect() hook to the App.js file so our app will fetch data only once when the App is loaded. You can check how I did that with loadRocketsThunk.

In requirements you will find this: 
"NOTE: Make sure you only dispatch those actions once and do not add data to store on every re-render (i.e. when changing views / using navigation)."